views/js/qtiCreator/tpl/notifications/invalidInfoBox.tpl

```diff
@@ -1,4 +1,4 @@
-<div class="feedback-warning popup" data-widget-component="{{serial}}">
+<div class="feedback-warning-popup" data-widget-component="{{serial}}">
```

Comment:
not sure that this a good idea: this box could be used by other interactions or anything in the qtiCreator context, could you do not change classes, you could access this.$messageBox and add there your own class declaration

Reply:
understood, you changed all states component, I will check it again